go-controller/pkg/ovn/policy.go

```diff
@@ -195,7 +195,7 @@ func (oc *Controller) updateStaleDefaultDenyACLNames(npType knet.PolicyType, gre
 	p := func(item *nbdb.ACL) bool {
 		return item.ExternalIDs[defaultDenyPolicyTypeACLExtIdKey] == string(npType) && // default-deny-policy-type:Egress or default-deny-policy-type:Ingress
 			strings.Contains(item.Match, gressSuffix) && // Match:inport ==	@ablah80448_egressDefaultDeny or Match:inport == @ablah80448_ingressDefaultDeny
-			!strings.Contains(*item.Name, arpAllowPolicySuffix) && // != name: namespace_ARPallowPolicy
+			!strings.Contains(item.Match, "arp") && // != match: (arp || nd)
```

Comment:
Something we forgot about the first time around, set the ACL UUID to make the operation more performant:
`newACL.UUID := aclList[0].UUID`

Reply:
oh! ack, so if I understand this correctly when I set the UUID while giving it for update, since UUID is a unique identifier, the indexing on it will help in having a more efficient fetchACL algorithm?